src/components/page/Pages.js

```diff
@@ -68,10 +69,17 @@ class Pages extends React.Component {
     return (
```

Comment:
`Pages` has currently more space between border and content than the `Page`.
adding class `content` to `hero-body` (Line 60) and removing class `container` from Line 61 would make `Pages` appear the same way as `Page`

Reply:
`has-text-centered` should be removed from L61 as well (CategorySidebar is otherwise centered)

Reply:
True.